doc/vagrant.md

```diff
@@ -0,0 +1,23 @@
+## Testing `kubeadm-nspawn` using the included Vagrantfile
```

Comment:
`Testing `kubeadm-nspawn` with [Vagrant](https://www.vagrantup.com/)`

With a small description...

A vagrant file is provided to test on various Linux distributions.